jenkins-vars.yml

```diff
@@ -3,7 +3,7 @@
 # jenkins variables
 project_name: docker-sabnzbd
 external_type: na
```

Comment:
Can this be GitHub now? Might need other changes.

Reply:
No, this is the pre-release one. We want it to build both pre-release and stable releases